common/source/docs/common-lua-scripts.rst

```diff
@@ -289,6 +366,35 @@ The relay library proivdes access to controlling relay outputs.
 
 - :code:`toggle(relay_num)` - Toggles the requested relay on or off.
 
+
+Servo_Channels (SRV_Channels:)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- :code:`find_channel(output function)` - Returns first servo output number -1 of an output assigned "output function" (See ``SERVOx_FUNCTION`` parameters ). False if none is assigned.
+
+Servo Output
+~~~~~~~~~~~~
+
+This is special case that is not generated via the binding methods.
+
+- :code:`servo.set_output (function number, PWM value)` -Sets servo outputs of type "function number" to PWM values between 1000 and 2000.
+
+
+Examples
+========
```

Comment:
This section is extremely low value. We should link to the examples folder, and include a single well annotated/explained script. This is what we do up in the sample script section, so just delete this whole area, improve that as needed, and link to the examples up there.

Reply:
sorry, disagree here....and link to examples was provided lower down

Reply:
The sample script is higher quality then the examples here though. It shows much the same thing, and has a description for every line, and includes demos of how you'd actually chain the functions together. The snippets here don't present anything new, and the comments on them make a lot less sense.

Reply:
removed section....personally it took me a while to understand the  useage, even looking at the examples folder....not being familiar with namespace conventions, etc...
I think that more examples in the examples folders are needed with better commenting in order for this to be useful to the average user...but will let you or others interested in promoting LUA scripting to a wider base do that...